esm-samples/jsapi-node/README.md

```diff
@@ -1,48 +1,47 @@
+Integrating Node.js with [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) can be done by building the app with [native ESM in a supported Node version](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html) or by transpiling to CommonJS. This sample contains examples of both approaches.
+
+## Working with assets
+
+Working with the ArcGIS API for JavaScript's ES modules in Node.js requires configuring the APIs assets, since the application will be running on a local server and not in a web app. The assets include styles, images, fonts, and localization files. Be sure to set [`config.assetsPath`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath) so that the assets are correctly resolved, for example:
 
 ```js
-// rollup.config.js
-{
-import commonjs from "@rollup/plugin-commonjs";
-import resolve from "@rollup/plugin-node-resolve";
-
-export default {
-  input: ["src/request.js", "src/projection.js", "src/webmap.js"],
-  output: {
-    chunkFileNames: "chunks/[name].js",
-    dir: "dist",
-    format: "cjs"
-  },
-  external: ["whatwg-fetch"],
-  plugins: [resolve(), commonjs()],
-  preserveEntrySignatures: false
-};
+import esriConfig from '@arcgis/core/config.js';
+esriConfig.assetsPath = "./public/assets"; // relative to when running in root
 ```
 
-Because Node does not have a native `fetch`, you will need to load a polyfill.
+An example can be found in [`projection.js`](https://github.com/Esri/jsapi-resources/blob/master/esm-samples/jsapi-node/src/projection.js#L6).
+
+## Polyfills
+
+Because Node does not have a native `fetch` or `abort-controller`, you will need to load polyfills:
 
 ```js
 require("cross-fetch/polyfill");
+require("abort-controller/polyfill");
 ```
 
-You can find native ESM samples in the [native-esm folder](./native-esm).
+An example can be found in the [webmap.js](https://github.com/Esri/jsapi-resources/blob/master/esm-samples/jsapi-node/webmap.js#L4-L5) file.
 
-Be sure to set [`config.assetsPath`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath) so that the assets are correctly resolved, for example:
+## Native ESM samples
 
-```js
-import esriConfig from '@arcgis/core/config.js';
-
-esriConfig.assetsPath = "node_modules/@arcgis/core/assets"; // relative to when running in root
-```
+You can find native ESM samples in the [/native-esm](./native-esm) folder.
 
 ## IdentityManager
 
-You will also want to disable the [`IdentityManager`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#request) so it doesn't attempt to load DOM related JavaScript.
+You will also want to disable [`IdentityManager`](https://developers.arcgis.com/javascript/latest/api-reference/esri-identity-IdentityManager.html) the using [`config.request`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#request) so it doesn't attempt to load DOM-related JavaScript.
```

Comment:
Typo? You will also want to disable IdentityManager _using_ config.request

Reply:
Typo, thanks.